services/site/src/pages/index.tsx

```diff
@@ -80,67 +68,78 @@ const Home: React.VoidFunctionComponent<HomeProps> = ({ fmType }) => {
         />
         <meta name="twitter:image" content="https://a11yphant.com/images/SEO/mockups-social-media.jpg" />
         {/* <!-- General Meta Tags --> */}
         <meta name="theme-color" content="#121212" media="(prefers-color-scheme: dark)" />
         <meta name="theme-color" content="#FFFFFF" media="(prefers-color-scheme: light)" />
       </Head>
-      <Navigation displayBreadcrumbs />
-      <main>
-        <h1 className={clsx("sr-only")} aria-label="Allyphant">
-          a11yphant
-        </h1>
-        <div className={clsx("w-full h-full")}>
-          {!currentUser?.isRegistered && <Hero />}
-          <section
-            id="challenges"
-            className={clsx("max-w-screen-3xl pt-10 mx-8 mt-22 mb-4", "sm:mx-12 sm:mt-18 sm:mb-12", "md:mx-24 md:mb-24", "2xl:mx-auto")}
-          >
-            <ChallengeHeader className={clsx("2xl:mx-24")} userLoggedIn={currentUser?.isRegistered} />
-
-            <Legend className={clsx("2xl:mx-24")} />
-            <p className={clsx("mb-12 text-grey-middle", "2xl:mx-24")}>
-              <span className="sr-only">Disclaimer:</span>
-              <span aria-hidden="true" className="text-xl text-grey-middle">
-                *
-              </span>{" "}
-              Currently, we only offer easy challenges. However, our small team is dedicated to producing more content for you. We will announce new
-              challenges on our{" "}
-              <InTextLink
-                href="https://twitter.com/a11yphant"
-                opensInNewTab
-                overrideClassName
-                className="text-grey-middle font-sans font-normal border-grey-middle
-        transition-colors duration-300
-        hover:text-primary-light hover:border-transparent
-        focus-rounded-instead-of-underline"
-              >
-                Twitter account
-              </InTextLink>
-              . <br />
-            </p>
-
-            {dataChallengesInProgress?.challenges.length > 0 && (
-              <ChallengeList
-                className={clsx("2xl:mx-24")}
-                heading={"Continue where you left"}
-                challenges={dataChallengesInProgress.challenges}
-                displayCompleted={false}
-              />
-            )}
-
-            {easyChallenges.length > 0 && (
-              <ChallengeList
-                className={clsx("2xl:mx-24")}
-                heading={dataChallengesInProgress?.challenges.length === 0 ? "All challenges" : "Other challenges"}
-                challenges={easyChallenges}
-              />
+      <Navigation />
+      <main className="bg-texture bg-repeat-y bg-contain bg-origin-border bg-top">
+        <div className={clsx("h-full max-w-screen-3xl px-8", "sm:px-12", "md:px-24", "xl:px-24", "2xl:mx-auto")}>
+          <HeroSection />
+          <MediaSection />
```

Comment:
The media section is too prominent. It should be either above or below "What others say about us"